README.md

```diff
@@ -1,15 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio
+# WebSummit
 
-> Microverse assigned project
+> Microverse First Capstone Project
 
 
 ## Built With
 
 - HTML, CSS, Javascript
 
-## Live Demo (https://habibpayenda.github.io/portfolio/)
+## Live Demo ()
```

Comment:
- [ ] Please make sure to add a live demo to your readme file as this is part of the requirements for this project, so please work on this issue.

Reply:
For sure!